src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```diff
@@ -344,16 +344,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Target Name="ResolvePackageDependenciesDesignTime"
           Returns="@(_PackageDependenciesDesignTime)"
-          DependsOnTargets="RunResolvePackageDependencies;ResolveAssemblyReferencesDesignTime">
-
-    <PreprocessPackageDependenciesDesignTime
```

Comment:
It seems that we cannot remove `PreprocessPackageDependenciesDesignTime`  since it is used by  `CollectSDKReferencesDesignTime`.

https://github.com/dotnet/sdk/blob/main/src/Tasks/Microsoft.NET.Build.Tasks/CollectSDKReferencesDesignTime.cs#L37

Reply:
done